### ISSUE TITLE
React Code not rendering on production

### ISSUE THREAD

**Issue by leonardolombardi07** [enuchi/React-Google-Apps-Script#138]
Hi! First I would like to thank you, this is an amazing repository. Thanks.

So, I must say I don't have much knowledge about Webpack and tooling as a whole, so I don't know how to debug my code in order to understand what's going on. Help needed 😅

When I give the command "npm start" and develop the extension in development mode, everything works perfectly. Now, when I give the command "npm run deploy" and the code for production is sent to Google App Script, _none of the UI is rendered in the Sidebar_ I developed. The server functions seem to work perfectly though (when I run them directly from Google App Script).

With "DynamicCdnWebpackPluginConfig" having the "verbose" property being true, I see the following logs:

✔️ 'react' will be served by https://unpkg.com/react@18.2.0/umd/react.production.min.js
✔️ 'react-dom' will be served by https://unpkg.com/react-dom@18.2.0/umd/react-dom.production.min.js
✔️ 'semantic-ui-react' will be served by https://unpkg.com/semantic-ui-react@2.1.3/dist/umd/semantic-ui-react.min.js
❌ 'gas-client' couldn't be found, please add it to https://github.com/mastilver/module-to-cdn/blob/master/modules.json
❌ 'uuid' couldn't be found, please add it to https://github.com/mastilver/module-to-cdn/blob/master/modules.json

I suspect it has something to do with the 'gas-client' library. This is the repository:
[https://github.com/leonardolombardi07/caption-manager](https://github.com/leonardolombardi07/caption-manager)

Any help is appreciated. Again, thanks!

**Comment by leonardolombardi07** [comment]
The best thing would be automatically detect the global exported variable from the CDN code, but I don't how hard this is. So maybe just improve a little bit the comment:

"var" is important to get right -- this should be the exposed global. Look up "webpack externals" for info.

To something like:
"var" is important to get right - this should be the exposed global from the CDN's code. 
You can find the name of this variable by looking for "exports" in the CDN code.
Look up "webpack externals" for info.

Not sure if that really helps more, but i think it would have helped me

**Comment by enuchi** [comment]
Yeah not sure. I may tweak the language a bit but it is just tricky to figure out regardless and some packages don't even expose the variable anyway. Have struggled with this myself to be honest.
